api/utils/sshutils/ssh.go

```diff
@@ -264,7 +264,7 @@ func hostKeyFallbackFunc(knownHosts []ssh.PublicKey) func(hostname string, remot
 
 // KeysEqual is constant time compare of the keys to avoid timing attacks
 func KeysEqual(ak, bk ssh.PublicKey) bool {
-	a := ssh.Marshal(ak)
-	b := ssh.Marshal(bk)
```

Comment:
This function expects a struct or pointer, but `ssh.ed25519Key` is a type alias for `[]byte`.